tokentrim/tokentrim.py

```diff
@@ -13,12 +13,9 @@
   'code-llama': 1048, # I'm not sure this is correct.
 }
 
-def num_tokens_from_messages(messages: List[Dict[str, Any]],
-                             model) -> int:
-  """
-  Function to return the number of tokens used by a list of messages.
-  """
+MAX_ITERATIONS = 12
 
+def get_encoding(model):
```

Comment:
Getting the encoder is now used in another place to compare against token length instead of character length, so I abstracted this out into a method we can call when we need to retrieve the encoder for the current model.